platform/android/java/lib/src/org/godotengine/godot/Godot.java

```diff
@@ -1014,4 +1032,12 @@ public void initInputDevices() {
 	private GodotRenderView getRenderView() { // used by native side to get renderView
 		return mRenderView;
 	}
+
+	@Keep
+	private void callIntentWithCommandLineArgs(String intentName, String[] args) throws ClassNotFoundException {
+		Class<?> c = Class.forName(intentName);
+		Intent intent = new Intent(getActivity(), c);
+		intent.putExtra(COMMAND_LINE_INTENT_NAME, args);
+		startActivity(intent);
```

Comment:
currently no way of communicating so no way of getting process ID (and if we can get Im not sure Android allows communication process wise...)

Reply:
Pass command line argument to the new godot instance

Reply:
The point is that creating a new process needs to be in a new activity, and I need the new activity to run while sending data back to the original activity (running a game needs to be able to talk to the editor for debugging/exit purposes)

Reply:
Addressed in https://github.com/thebestnom/godot/pull/2